Lesson_5/Task_2.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task_2</title>
+    <style>
+        th {
+            color: #4566af;
+            text-align: left;
+            text-indent: 15px;
+            height: 40px;
+            border-bottom: 2px dotted #4566af;
+            font-size: 26px;
+        }
+        td {
+            color: #9f8bb0;
+            text-align: left;
+            text-indent: 15px;
+            height: 40px;
+            font-size: 22px;
+        }
+    </style>
+</head>
+<body style="font-family: Calibri, SansSerif;">
+    <div style="margin: 100px 100px">
+<!--Я обернул всё в div с маржином, потому что таблица от края до края выглядит по-уродски :)-->
+        <table style="border: 2px solid #4566af; width: 100%; table-layout: fixed; min-width: 600px">
+            <tr>
+                <th style="width: 400px">Company</th>
+                <th>Q1</th>
+                <th>Q2</th>
+                <th>Q3</th>
+                <th>Q4</th>
+            </tr>
+            <tr>
+                <td style="width: 300px">Microsoft</td>
```

Comment:
I think, you can set width: 400px only for first column

Reply:
done